<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" hash="2845225252">
    <data-source source="LOCAL" name="Django default" uuid="1c42fadf-393c-4948-84ee-019804127393">
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Python27\example\example\test.db</jdbc-url>
      <libraries>
        <library>
          <url>file://$USER_HOME$/.PyCharm30/config/jdbc-drivers/xerial-sqlite-license.txt</url>
        </library>
        <library>
          <url>jar://$USER_HOME$/.PyCharm30/config/jdbc-drivers/sqlite-jdbc-3.7.6.3-20110609.081603-3.jar!/</url>
        </library>
      </libraries>
    </data-source>
    <data-source source="LOCAL" name="Django followup" uuid="33bf077d-3924-4146-a410-3f711540d052">
      <jdbc-driver>com.mysql.jdbc.Driver</jdbc-driver>
      <jdbc-url>jdbc:mysql://imagedbinstance.ckjalzbdlxio.us-east-1.rds.amazonaws.com:3306/NLDB</jdbc-url>
      <user-name>awsimagedb</user-name>
      <user-password>dfcbdfdddfd9dfdadfcbdfd9dfd9dfdddfc5dfd8dfce</user-password>
      <libraries>
        <library>
          <url>file://$USER_HOME$/.PyCharm30/config/jdbc-drivers/mysql-connector-license.txt</url>
        </library>
        <library>
          <url>file://$USER_HOME$/.PyCharm30/config/jdbc-drivers/mysql-connector-java-5.1.26-bin.jar</url>
        </library>
      </libraries>
    </data-source>
  </component>
</project>

